.rubocop.yml

```diff
@@ -8,6 +8,7 @@ AllCops:
     - 'config/initializers/devise.rb'
     - 'db/migrate/20160405212342_initial_schema.rb'
   UseCache: true
+  TargetRubyVersion: 2.3.1
```

Comment:
This is what is causing code climate to fail.. did you have to add this for this change?